### ISSUE TITLE
No file actually created an no preview

### ISSUE THREAD

**Issue by dudiyalov** [coleam00/bolt.new-any-llm#187]
### Describe the bug

Using:
- docker
- chrome canary
- ANTHROPIC_API_KEY
- GROQ_API_KEY
- GOOGLE_GENERATIVE_AI_API_KEY

In Anthropic seems to do the job but no preview
In Google of files and no preview
In Grok it do nothing

### Link to the Bolt URL that caused the error

http://localhost:5173/

### Steps to reproduce

Start the app enter a prompt seems to run write lots of stuff but no preview and no files in the directory

### Expected behavior

It dose not help if the is no preview and can not work on any project

### Screen Recording / Screenshot

![image](https://github.com/user-attachments/assets/b0600793-3576-4035-b9e4-6d523d5460fc)

### Platform

- OS: Windows + Docker desktop
- Browser: Chrome
- Version: canary

**Comment by DavidWalters123** [comment]
I'm having the same issue here

**Comment by aithunderbolt** [comment]
I'm facing the same issue running on Windows and no docker.
Browser: Chrome

**Comment by xilorez-ux** [comment]
Try with different browsers. If this doesn't work, that means there is a specific parameter blocked by your computer for unauthorized execution for somewhat reason, or stuff like that.
At least if you try with other browsers we'll be able to have a better undesrtanding of what's specifically happening...
If you've already done this, please let us know the result.

**Comment by xilorez-ux** [comment]
> ![image](https://private-user-images.githubusercontent.com/100776531/383593407-60f5f063-dfc4-424e-b96a-b554250d85b4.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzA5MDM1OTEsIm5iZiI6MTczMDkwMzI5MSwicGF0aCI6Ii8xMDA3NzY1MzEvMzgzNTkzNDA3LTYwZjVmMDYzLWRmYzQtNDI0ZS1iOTZhLWI1NTQyNTBkODViNC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMTA2JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTEwNlQxNDI4MTFaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT02ODIxNmFkZmU3NjAxODEwMjk1MzIyZjhiMjViMGU2OTlkYWIzY2Q2OWQ5M2QwN2RjYzRkYjNiZWFkNjFiZmM0JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.uru8Z4vpjCtzEDn3Q4Pn5nKP8H_KdU7MyyZKa0sRkY4)
> 
> I have facing the above issue. i get a 404 error in the webcontainer i have no clue on this as this i alll automated. I am running local ollama model

It seems like the issue comes from unauthorized connection for somewhat reason.

If you're using docker, maybe try disabling your firewall, as it is probably a closed port issue.

If you're not using docker, try running it from a terminal :
- go in the folder where you installed the project, copy the file path
- open terminal from windows search bar, run it as administrator
- type 'cd %your-file-path%' (remove quote marks if you pasted it)
- then follow the steps below :

![image](https://github.com/user-attachments/assets/55c24532-b316-4f84-8de1-63425ef3ff8b)

If the 'run dev' fails, also try without firewall

**Comment by dudiyalov** [comment]
Still error also try to put it with a directory with all permission. Now it dose not un at all
![image](https://github.com/user-attachments/assets/df145b69-d9a2-49ca-9759-516270bf2b83)

**Comment by xilorez-ux** [comment]
> Still error also try to put it with a directory with all permission. Now it dose not un at all ![image](https://private-user-images.githubusercontent.com/72905875/383613025-df145b69-d9a2-49ca-9759-516270bf2b83.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzA5MDg4ODAsIm5iZiI6MTczMDkwODU4MCwicGF0aCI6Ii83MjkwNTg3NS8zODM2MTMwMjUtZGYxNDViNjktZDlhMi00OWNhLTk3NTktNTE2MjcwYmYyYjgzLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDExMDYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMTA2VDE1NTYyMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWRlODg0ZjZhZTQxNTBlYTdmMzNiMzMzNjYxNzAyYjZiNGRjMzk4MDQzZGViNzA0NDYxOWI0ZDc0ZGEwMTNkMjMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.t1KfMwogQd3aWDUt0IkSWCnIDkoEXy3AaApd2vMXgtM)

Yeah sorry I looked if there were similar problems on reddit or so but didn't find anything, I have no idea. Hopefully someone else will be able to tell us what's going on here... Also if you find a solution let us know

**Comment by patrykwegrzyn** [comment]
Same issue here getting 404 NOT FOUND for webcontainer preview links

**Comment by tapankumarpatro** [comment]
Hi team, same issue, my setup

Mac with Docker, running in localhost  
Tried with different browser, Code is getting generated but not able to see any preview of it
<img width="1742" alt="image" src="https://github.com/user-attachments/assets/dff6e04f-cd7d-4caf-b2f9-01e39e57e3d2">

<img width="1545" alt="image" src="https://github.com/user-attachments/assets/51c07044-253d-4792-aa1f-b897c9c5ed3c">

I am new to web dev so didn't understand these logs would be great if someone can help
<img width="781" alt="image" src="https://github.com/user-attachments/assets/580b9b03-fef0-496c-9e76-35ff9a76d782">
